.travis.yml

```diff
@@ -11,23 +11,40 @@ branches:
   only:
   - master
 
-# install proper NodeJS for bids-validator 
 before_install:
+  # install spine-generic for acquisition parameters check
+  # TODO: we should probably version-pin this?
+  - pip install spine-generic@git+https://github.com/kousu/spine-generic.git@patch-1
```

Comment:
don't we want to point to spine-generic/spine-generic instead?

Reply:
ok i'm just seeing this https://github.com/spine-generic/data-multi-subject/pull/6#issuecomment-674655514